src/components/NavSection.vue

```diff
@@ -1,42 +1,50 @@
 <template>
-  <nav :aria-label="$t('header.aria.primary')" class="navbar">
-    <div class="navbar-brand text-white">
-      <!-- eslint-disable @intlify/vue-i18n/no-raw-text -->
-      <NuxtLink class="logo" :to="localePath('/')">
-        <img
+  <!-- Refer to the Bulma docs for markup: https://bulma.io/documentation/components/navbar/ -->
+  <nav
+    role="navigation"
+    class="navbar embedded"
+    :aria-label="$t('header.aria.primary')"
+  >
+    <div class="navbar-brand flex-grow md:flex-grow-0">
+      <NuxtLink
+        to="/"
+        class="navbar-item"
+        style="align-self: center; line-height: 0"
+      >
+        <!-- width and height chosen w.r.t. viewBox "0 0 280 42" -->
+        <OpenverseLogo
+          :style="{ width: '160px', height: '24px' }"
```

Comment:
The reason for setting the width and height _HTML properties_ (not inline style) here was to minimize CLS as described here: https://web.dev/optimize-cls/#images-without-dimensions

Reply:
Nice. I wasn't aware of their utility. Restored in 072672c.

Reply:
Should we remove the inline styles now or are both necessary?

Reply:
You're right that the `style` attr is no longer necessary. Removed in ad5a6ed.